src/components/SolWarning/style.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import { colors, typography } from '@static/theme'
+
+const useStyles = makeStyles((theme: Theme) => ({
+  warningContainer: {
```

Comment:
i don't think it is necessary, we already have ufils for bluring backgrounds, uses in our modals, check it